src/cpp/desktop/html/error.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
+        <style>
+            h1 {
+                margin-left: auto;
+                margin-right: auto;
+            }
+
+            .error {
+                color: red;
```

Comment:
Red on #f0f0f0 does not have sufficient contrast and fails WCAG AA and AAA validation. This gives 3.5:1 ratio, needs to be minimum 4.5:1 for normal text.

![2019-10-07_16-05-40](https://user-images.githubusercontent.com/10569626/66355496-5cfdbe00-e91c-11e9-9abb-c18d6eb4339c.png)


https://webaim.org/resources/contrastchecker/

Reply:
Switched to #db0000 for > 4.5 contrast ratio.